sdk/src/tx/types.ts

```diff
@@ -72,3 +66,9 @@ export class TxSendError extends Error {
 		}
 	}
 }
+
+export type BlockHashAndValidSlot = {
```

Comment:
should rename this slot -> blockHeight because they're not the same. Also web3.js has an identical type `BlockhashWithExpiryBlockHeight`